core/web/router.go

```diff
@@ -364,11 +364,22 @@ func v2Routes(app chainlink.Application, r *gin.RouterGroup) {
 		authv2.PATCH("/chains/evm/:ID", chc.Update)
 		authv2.DELETE("/chains/evm/:ID", chc.Delete)
 
-		nc := NodesController{app}
-		authv2.GET("/nodes", paginatedRequest(nc.Index))
-		authv2.GET("/chains/evm/:ID/nodes", paginatedRequest(nc.Index))
-		authv2.POST("/nodes", nc.Create)
-		authv2.DELETE("/nodes/:ID", nc.Delete)
+		enc := EVMNodesController{app}
+		// TODO still EVM only https://app.shortcut.com/chainlinklabs/story/26276/multi-chain-type-ui-node-chain-configuration
+		authv2.GET("/nodes", paginatedRequest(enc.Index))
+		authv2.POST("/nodes", enc.Create)
+		authv2.DELETE("/nodes/:ID", enc.Delete)
+
+		authv2.GET("/nodes/evm", paginatedRequest(enc.Index))
```

Comment:
This namespacing seems the wrong way around, it should probably reflect the command structure like: `/evm/nodes` and `/terra/nodes`.

Reply:
OK this makes sense.